### ISSUE TITLE
Proposal to Change the Context Entry for "tags"

### ISSUE THREAD

**Issue by melvincarvalho** [nostrcg/schema#2]
Propose a modification to the current JSON-LD context for the "tags" field in our data model. This change is suggested to better align our model with the commonly accepted best practices for handling arrays or lists of data in JSON-LD.

### Current Implementation:

In the present context, the "tags" field is defined as follows:

```json
"tags": {
  "@id": "nostr:tags",
  "@type": "xsd:string"
}
```

This configuration treats "tags" as a string, which might limit the expressivity of our data model, especially when we need to represent multiple tags associated with a single entity.

### Proposed Change:

We propose to modify the context entry for "tags" to use `@container": "@list"` as per the suggestion in the [W3C issue](https://github.com/w3c/vc-data-model/issues/1132#issuecomment-1550589864). This change would allow us to represent "tags" as an array of strings instead of a single string. 

```json
"tags": {
  "@id": "nostr:tags",
  "@container": "@list"
}
```

This modification would provide a more flexible and accurate representation of the "tags" data, allowing us to handle multiple tags for a single entity.

### Impact and Benefits:

This change should significantly improve our ability to represent, query, and manipulate data that includes multiple tags.

In conclusion, adopting this change would bring us in line with widely-adopted best practices for JSON-LD modeling, improve the robustness and expressivity of our data model, and provide better support for handling arrays or lists of data.
